code/index.html

```diff
@@ -2,21 +2,79 @@
+        <a class="nav-link" href="About">About</a>
+        <a class="nav-link" href="Search">Search</a>
+        <a class="nav-link" href="contact">Contact</a>
+      </nav>
     </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+      
+    <!-- The big news content -->
+    <section class="big-news-container">
+      <article>
+       <h2 class="big-news-title">Politics</h2>
+        <div class="big-news-items">
+        <div class="img-big-news">
+          <img src="./USA.jpg" alt="Bild på en stor nyhet" width="500px">
+        </div>
+        <p class="big-news-text">"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</p>
+      </div>
+      </article>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <!-- The other news  -->
+    <section class="small-news-container">
+       
+      <article class="small-news">
+        <h3 class="small-news-title">Climate</h3>
+        <img class="img-small-news" src="./Taj-mahal.jpg" alt="Picture of Taj-mahal" width="300px">
+      </article>
+
+      <article class="small-news">
+        <h3 class="small-news-title">Sustainability</h3>
+        <img class="img-small-news" src="./Fish.jpg" alt="Picture of fishes" width="300px">
+        </article>
+
+      <article class="small-news">
+        <h3 class="small-news-title">Humanity</h3>
+        <img class="img-small-news" src="./Argentina.jpg" alt="Picture of Argentinas flag" width="300px">
+      </article>
+
+      <article class="small-news">
+        <h3 class="small-news-title">Animals</h3>
+        <img class="img-small-news" src="./Elephants.jpg" alt="Picture of elephants" width="300px">
+      </article>
+
+      <article class="small-news">
+        <h3 class="small-news-title">Health</h3>
+        <img class="img-small-news" src="./No-smoking.jpg" alt="Picture of no-smoking sign" width="300px">
+      </article>
+
+      <article class="small-news">
+        <h3 class="small-news-title">Humanity</h3>
+        <img class="img-small-news" src="Women.jpg" alt="Picture of women" width="300px">
+      </article>
+
+      <article class="small-news">
+        <h3 class="small-news-title">Sustainability</h3>
+        <img class="img-small-news" src="Power-plant.jpg" alt="Picture of power-plant" width="300px">
+      </article>
+
+      <article class="small-news">
+        <h3 class="small-news-title">Environment</h3>
+        <img class="img-small-news" src="marine-life.jpg" alt="Picture of marine-life" width="300px">
+      </article>
+
     </section>
   </body>
 </html>
```

Comment:
Very good structure of your HTML file! The correct indentation makes it easy to follow 👍